src/nvim/api/ui_events.in.h

```diff
@@ -0,0 +1,70 @@
+#ifndef NVIM_API_UI_EVENTS_IN_H
+#define NVIM_API_UI_EVENTS_IN_H
+
+// This file is not compiled, just parsed for definitons
+#ifdef INCLUDE_GENERATED_DECLARATIONS
+#error "don't include this file, include nvim/ui.h"
+#endif
+
+#include "nvim/api/private/defs.h"
+#include "nvim/func_attr.h"
+#include "nvim/ui.h"
+void resize(Integer rows, Integer columns)
+    FUNC_API_SINCE(2);
+void clear(void)
+    FUNC_API_SINCE(2);
+void eol_clear(void)
+    FUNC_API_SINCE(2);
+void cursor_goto(Integer row, Integer col)
+    FUNC_API_SINCE(2);
```

Comment:
I don't think it meaningful to track versions earlier than the versioning mechanism itself. So actually it should be 3 as that will be the first version with events in the frozen metadata